### PR TITLE
New onboarding modal

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
We've redesigned the onboarding modal to have some more detail

## Description
This PR updates the onboarding modal to have 7 views and supports translations for EN, DE and FR.

Existing translations are used for the unchanged modal views, this does lead to some cases where we have one modal with a mix of two languages i.e. Spanish is selected and a translation for the title is included by not the body text. 

Image sizes and compression could be improved, mobile image positioning could also be improved, feel free to give feedback.

### Preview

image
### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
Do you want me to add webp versions of the gifs?

**Comment by tonypls** [comment]
> Do you want me to add webp versions of the gifs?

Yeah that would be great

**Comment by VIKTORVAV99** [comment]
> > Do you want me to add webp versions of the gifs?
> 
> Yeah that would be great

Will do that before the end of the day. 👍🏻

**Comment by VIKTORVAV99** [comment]
Added the webp versions of all the images but I had to use some extreme compression options to get a good size saving over the gif file so the quality might be worse so feel free to discard that one.

The webp versions of the pngs should be pretty straight forward to use though.

File: web/public/locales/en.json [hunk 1]
```diff
@@ -27,17 +27,30 @@
...
     "view4": {
+      "header": "Switch between different views",
+      "text": "Toggle between Electricity Production and Consumption, as well as Country or Zone views and discover real-time insights into global energy dynamics and make informed choices for a sustainable future."
+    },
+    "view5": {
+      "header": "Switch between electricity and emissions views",
+      "text": "Toggle to alternate between charts that display a zone's electricity information or carbon emissions. The values shown represent the total for the selected period, and the information aligns with the chosen production or consumption mode."
+    },
+    "view6": {
+      "header": "Explore past situations",
```
Comment (madsnedergaard) [inline web/public/locales/en.json:50]: Honestly don't really think this title is great ¯\\\_(ツ)\_/¯

**Review by madsnedergaard — APPROVED** [review]
Nice, a few minor comments, but otherwise it looks good to me. The gif is pretty messy grainy :D

File: web/public/images/onboarding/splitLayers.png
[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply web/public/images/onboarding/splitLayers.png:1]: Could we add the same transparency to the close button as we have done to the other icons? that could help with the issue somewhat?

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM!

But maybe we should dump the animated webp version...

**Pull request merged**